services/src/main/java/org/jboss/windup/web/services/model/AnalysisContext.java

```diff
@@ -273,6 +276,21 @@ public void removeApplication(RegisteredApplication application)
         this.getApplications().remove(application);
     }
 
+    public AnalysisContext clone()
+    {
+        AnalysisContext clone = new AnalysisContext(this.migrationProject);
```

Comment:
Well, I didn't realize I could use that :) That is basically the only reason, why it is done this way.